### ISSUE TITLE
Fabric_DetailsList : Screen reader is narrating verbose information while navigating through table elements.

### ISSUE THREAD

**Issue by v-vasud** [OfficeDev/office-ui-fabric-react#8312]
### Environment Information 
OS: Windows 10
Version: 1809(OS Build 18298.1000)
Browser: Microsoft Edge (44.18298.1000.0)
Screen Reader: Narrator

#### User Experience:
Users who rely on screen reader are getting affected as narrator is reading inappropriate information when navigating through table.

#### Reproduction Steps:
1) Turn on Narrator and Open https://uifabric-prod.azurewebsites.net/?example=jhkjhkj#/components/detailslist#Implementation URL in Edge browser
2) "DetailsList Implementation" page will be displayed
3) Navigate to table elements in scan mode and observe whether screen reader is narrating any verbose information.

#### Actual behavior:

Narrator is announcing state of row as not-selected even though we don’t have any control.

#### Expected behavior:

Narrator should not announce verbose information

#### Priorities and help requested:

Requested priority: (High) 
 
Products/sites affected: M365 Security and compliance

Impacted bugs due to this fabric issue
[Bug 1](https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fo365exchange.visualstudio.com%2FIP%2520Engineering%2F_workitems%2Fedit%2F962034&data=02%7C01%7Cv-bhachu%40microsoft.com%7C280db32e182f484bf29308d6a84821ed%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636881425098158861&sdata=nQ9JbaxJbcxz%2FJaqUdMlVCheUCNjHWVU30cEzwl8UCc%3D&reserved=0) : Narrator is reading inappropriate information when navigating through table.

**Comment by KevinTCoughlin** [comment]
@v-vasud I'm unable to navigate to the view you have screenshotted above.  However, given my analysis of your other reports i.e. https://github.com/OfficeDev/office-ui-fabric-react/issues/8310 I'm thinking that list is not an OUFR control.

Can you clarify how to reach that view if you firmly believe the list is a Fabric React control in this Angular application? I do not see "Monitoring & reports" upon login.

**Comment by KevinTCoughlin** [comment]
@v-vasud is your reference to `DetailsList` announcing "inappropriate verbose information" encapsulated by the following:

That the `DetailsList` example on https://developer.microsoft.com/en-us/fabric#/components/detailslist with "Enable modal selection" **disabled** reads the following per row focus:

```shell
one file iconIncididunt ut.one 5/1/2014 Labore Et 94 KB, non-selected, 
```

Specifically: "non-selected"

@v-vasud the answer to the above is critical in assessing what the fix and completion criteria is for this issue. If there is more "inappropriate verbose information" I should be aware of while addressing, please share.

**Comment by KevinTCoughlin** [comment]
This one I need to investigate as to whether or not we can omit `non-selected` / `selected` entirely when not in a selection mode.

**Comment by KevinTCoughlin** [comment]
I looked into this briefly and attempted to remove `aria-selected` and `aria-checked` from the `DetailsList` row and check components respectively. However, the rows still receive focus and announce their "selected" state. 

We need to confirm that removal of that announcement in the case where no selection is possible is desirable before proceeding. I'm not sure it is given that the rows continue to receive focus.

cc: @natalieethell

**Comment by xugao** [comment]
opened bug with narrator team [here](https://microsoft.visualstudio.com/OS/_workitems/edit/24928067)

**Comment by cschlechty** [comment]
I just tried and couldn't repro. Closing.
